### DISCUSSION TITLE
mongodb string is invalid

### DISCUSSION THREAD

**Discussion by Boby900** [prisma/prisma#21788] [Q&A] [answered]
### Bug description

first I ran the `npx prisma init` then after that as I am running the `npx prisma db pull` I am getting the error ```The provided database string is invalid.``` though I have checked my mongodb URI which is correct as I am using this same URI in other places as well.
I am new to Prisma
Any help would be highly appreciated.

### How to reproduce

first add the mongodb URI in the *.env*
secondly run the `npx prisma init`
then run `npx prisma db pull`

### Expected behavior

expected to build new schema

### Prisma information

NA
![Screenshot (195)](https://github.com/prisma/prisma/assets/111265239/7e7164f0-1f99-4f74-bffd-2b6f4a3b6c8d)

### Environment & setup

- OS:  Windows
- Database: MongoDB 
- Node.js version: v21.1.0

### Prisma Version

prisma                  : 5.5.2
@prisma/client          : 5.5.2
Current platform        : windows
Query Engine (Node-API) : libquery-engine aebc046ce8b88ebbcb45efe31cbe7d06fd6abc0a (at node_modules\@prisma\engines\query_engine-windows.dll.node)
Schema Engine           : schema-engine-cli aebc046ce8b88ebbcb45efe31cbe7d06fd6abc0a (at node_modules\@prisma\engines\schema-engine-windows.exe)
Schema Wasm             : @prisma/prisma-schema-wasm 5.5.1-1.aebc046ce8b88ebbcb45efe31cbe7d06fd6abc0a
Default Engines Hash    : aebc046ce8b88ebbcb45efe31cbe7d06fd6abc0a
Studio                  : 0.494.0

**Comment by janpio** [comment, marked as answer]
What does your connection string look like? Per the error message it does not contain a database name - which is needed for Prisma (but not for some other mongoDB tooling)

**Reply by Boby900** [reply]
DATABASE_URL= `mongodb+srv://bobyt2265:<password>@cluster0.eujid9v.mongodb.net/?retryWrites=true&w=majority`

I am replacing **password** with my mongodb password

**Reply by Boby900** [reply]
thanks for getting back to me @janpio

**Reply by Boby900** [reply]
I was reading this docs but still not able to figure out what wrong am I doing here

https://www.prisma.io/docs/concepts/database-connectors/mongodb

**Comment by Boby900** [comment]
thanks @janpio 
I was not adding the name of my db in URL, adding that fixed it
